app/controllers/notices_controller.rb

```diff
@@ -64,29 +55,30 @@ def display
     @notice.display(true)
   end
 
-  # Action to show profile picture only already authenticated
+  # Action to show notice picture
   def image
     if @notice.image?
-      if(params[:style] == "original" || params[:style] == "large" || params[:style] == "small")
-        send_file(@notice.image.path(params[:style]), filename:@notice.image_file_name, type: "image/jpg",disposition: 'inline',x_sendfile: true)
+      if (params[:style] == "original" || params[:style] == "large" || params[:style] == "small")
+        send_file(@notice.image.path(params[:style]), filename: @notice.image_file_name, type: "image/jpg", disposition: 'inline', x_sendfile: true)
```

Comment:
Line is too long. [148/100]<br>Prefer single-quoted strings when you don't need string interpolation or special symbols.